examples/demo.css

```diff
@@ -78,7 +78,7 @@ h1, h2, .section h3 {
     margin: auto;
     width: 80%;
     display: flex;
-    flex-wrap: wrap;
+    flex-wrap: nowrap;
```

Comment:
Hey @govindgoel, thanks for opening this...why did you change this to `nowarap`?

Reply:
Wrap is something like this

![Screenshot from 2020-02-16 22-57-07](https://user-images.githubusercontent.com/52847415/74609444-fbdac800-510f-11ea-8fec-b2e11907887c.png)